lib/bank.rb

```diff
@@ -0,0 +1,41 @@
+puts 'Your current balance is?'
+balance = gets.chomp.to_i
+
+done='no'
```

Comment:
Similar to the guessing game challenge, should've used a true and false check instead